src/transactions/transactionBuilder/signingMessage.ts

```diff
@@ -60,18 +59,6 @@ export function generateSigningMessage(bytes: Uint8Array, domainSeparator: strin
   return mergedArray;
 }
 
-/**
- * Generates the 'signing message' form of a serilizable value. It bcs serializes the value and uses the name of
- * its constructor as the domain separator.
- *
- * @param serializable An object that has a bcs serialized form
- *
- * @returns The Uint8Array of the signing message
- */
-export function generateSigningMessageForSerializable(serializable: Serializable): Uint8Array {
```

Comment:
lets add @deprected label and ask to use cryptoHasher class, then we will remove it in the next release